### PR TITLE
YOCS Teleop

### PR DESCRIPTION
**Pull request by bit-pirate**
This PR adds keyboard and joystick teleop tools.

`keyop is a modified version of`kobuki_keyop`, which originates from Yujin's propritary`ycs_keyop`.`yocs_joyop`has been directly migrate from`ycs_joyop`.

Soon or later we should drop `kobuki_keyop` and make use of both `yocs_keyop` and `yocs_joyop` for Kobuki (and Turtlebot).

### CODE REVIEW

**Comment by bit-pirate** [comment]
@jihoonl could you review (and merge) this PR?

File: yocs_keyop/src/keyop.cpp [hunk 1]
```diff
@@ -0,0 +1,438 @@
...
+#include "yocs_keyop/keyop.hpp"
+
+#define KEYCODE_RIGHT 67 // 0x43
+#define KEYCODE_LEFT  68 // 0x44
+#define KEYCODE_UP    65 // 0x41
+#define KEYCODE_DOWN  66 // 0x42
+#define KEYCODE_SPACE 32 // 0x20
+
+
+namespace yocs_keyop
```
Comment (jihoonl) [inline yocs_keyop/src/keyop.cpp:19]: Half packages in the yujin_ocs use `yocs` as namespace and half packages use its own namespace. How do you think about merging all into yocs namespace?

**Comment by jihoonl** [comment]
:smile_cat:

File: yocs_keyop/src/keyop.cpp (hunk 1 above)
Comment (bit-pirate) [inline yocs_keyop/src/keyop.cpp:19]: We are following up on this matter here: https://github.com/yujinrobot/yujin_ocs/issues/45

**Pull request merged**